src/oop/special_class_method/medium/q11/MainTest.java

```diff
@@ -0,0 +1,187 @@
+package oop.special_class_method.medium.q11;
+
+import global.ObjectTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.tools.CustomAssertions._assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MainTest {
+    // Java
+
+    private final String className = "Oven";
+    private final String attributeName0 = "currTemp";
+    private final String getAttributeMethodName0 = "getCurrTemp";
+    private final String setAttributeMethodName0 = "setCurrTemp";
+    private final String attributeName1 = "size";
+    private final String getAttributeMethodName1 = "getSize";
+    private final String attributeName2 = "colour";
+    private final String getAttributeMethodName2 = "getColour";
+    private final String methodName1 = "powerDecrease";
+    private final String methodName2 = "powerIncrease";
+    private final String methodName3 = "changePower";
+    private final int changeValue = 10;
+    private final int changeLimit = 100;
+
+    private ObjectTest testClass;
+
+    @BeforeEach
+    public void setUp() throws Throwable {
+        String classString = "oop.special_class_method.medium.q11." + className;
+        this.testClass = new ObjectTest(classString);
+        String modifiedClassMessage =
+                "You have modified the provided portions of class " + className + ". Please revert them to the original state.";
```

Comment:
This should have a clearer distinction between which parts of the class have been modified